fares-data-build-tool/fdbt-netex-output
# Implementation to introduce the return circular journey

# Description

This ticket deals with the non circular return single service (https://infinityworks.atlassian.net/browse/TFN-381)

The changes have utilised the Single Service generator code and the xml template.

# Testing instructions
- Check this branch locally 
- Also checkout https://github.com/fares-data-build-tool/fdbt-dev/pull/3
- You need to get the docker containers running locally so just run 'make' under the fdbt-dev branch
- Open another terminal session for fdbt-dev branch and run the following:
    -  `make generate-return-service`
    - Once done run the following command `awslocal s3api list-objects --bucket fdbt-unvalidated-netex-data-dev`
    - The above command will show the list of objects for the s3 bucket e.g. 
```
 {
            "Key": "Blackpool_Transport_11_2020-05-12T09:52:51.244Z.xml",
            "LastModified": "2020-05-12T09:52:51.286Z",
            "ETag": "\"e59dada4852d38aa1eb1e0fbaf19f22c\"",
            "Size": 88038,
            "StorageClass": "STANDARD",
            "Owner": {
                "DisplayName": "webfile",
                "ID": "75aa57f09aa0c8caeab4f8c24e99d10f8e7faeebf76c078efc7c6caea54ba06a"
            }
        }
```
   - Extract the key which is the filename
   - Run awslocal s3api get-object --bucket fdbt-unvalidated-netex-data-dev --key <ADD KEY EXTRACTED FROM ABOVE>  ~/Downloads/geo-monday-2.xml (this xml file can be renamed if you want to)
   - Once done then open Oxygen editor and open the xml file as named above
   - Run the custom validation option and make sure the custom url is configured to http://netex.uk/netex/schema/1.09c/xsd/NeTEx_publication.xsd
   - Validation should be successful

We also need to test the PointToPoint generator to see if all is ok. So would need to do the above again but run `make generate-point-to-point`

# Type of change


# Checklist:
